wagtail/tests/test_page_queryset.py

```diff
@@ -919,6 +919,23 @@ def test_specific_query_with_annotation(self):
         self.assertEqual(results.first().subscribers_count, 1)
         self.assertEqual(results.last().subscribers_count, 1)
 
+    def test_specific_query_with_alias(self):
+        # #11285
+        # Ensure alias() works with specific() queries.
+
+        pages = Page.objects.live()
+        user = self.create_test_user()
+        pages.first().subscribers.create(user=user, comment_notifications=False)
+        pages.last().subscribers.create(user=user, comment_notifications=False)
```

Comment:
I've just copied some stuff from the other tests but this is not really relevant. Happy to simplify to avoid confusion? What do we think?

Reply:
that's fine.